app/controllers/api/v1/service_plans_controller.rb

```diff
@@ -4,8 +4,20 @@ class ServicePlansController < ApplicationController
       include Api::V1::Mixins::IndexMixin
 
       def index
-        so = Catalog::ServicePlans.new(params.require(:portfolio_item_id))
-        render :json => so.process.items
+        portfolio_item = PortfolioItem.find(params.require(:portfolio_item_id))
+        Catalog::ImportServicePlans.new(portfolio_item.id).process if portfolio_item.service_plans.empty?
```

Comment:
@lindgrenj6 Should this be happening only if the user wants to override the service plan parameters. If they are content with the default service plan why would we import it into Catalog?

Reply:
That's a good point - I'll change it so `GET /portfolio_items/:id/service_plans` will check to see if we have an overwritten record otherwise reach out to topo to get a fresh one. 

Then whenever the user wants to edit a survey, I'll add `POST /service_plans {:portfolio_item_id => id}` to create a record and import the service plans so they can go ahead and make their edits.